tasks/main.yml

```diff
@@ -27,4 +27,4 @@
     dest: /etc/apt/preferences.d/deb_nodesource_com_node.pref
 
 - name: Install Node.js
-  apt: pkg=nodejs={{ nodejs_version }}.* state=installed update_cache=yes
+  apt: pkg=nodejs state=installed update_cache=yes
```

Comment:
the issue i ran into was `pkg=nodejs=0.10.*` was working, but `pkg=nodejs=4.x.*` was not working. haven't done much experimenting other than removing the version assertion.

btw, thanks for sharing this role, I appreciate it. :smile: